sdk-lib/src/main/java/cash/z/ecc/android/sdk/model/Account.kt

```diff
@@ -1,12 +1,117 @@
 package cash.z.ecc.android.sdk.model
 
+import cash.z.ecc.android.sdk.internal.jni.JNI_ACCOUNT_SEED_FP_BYTES_SIZE
+import cash.z.ecc.android.sdk.internal.model.JniAccount
+
 /**
- * A [ZIP 32](https://zips.z.cash/zip-0032) account index.
+ * Unique identifier for a specific account tracked by a `Synchronizer`.
+ *
+ * Account identifiers are "one-way stable": a given identifier always points to a
+ * specific viewing key within a specific `Synchronizer` instance, but the same viewing
+ * key may have multiple account identifiers over time. In particular, this SDK upholds
+ * the following properties:
+ *
+ * - When an account starts being tracked within a `Synchronizer` instance, a new
+ *   `Account` is generated.
+ * - If an `Account` is present within a `Synchronizer`, it always points to the same
+ *   account.
+ *
+ * What this means is that account identifiers are not stable across "wallet recreation
+ * events". Examples of these include:
+ * - Restoring a wallet from a backed-up seed.
+ * - Importing the same viewing key into two different wallet instances.
  *
- * @param value A 0-based account index.  Must be >= 0.
+ * @param accountUuid The account identifier.
+ * @param ufvk The account's Unified Full Viewing Key, if any.
+ * @param accountName A human-readable name for the account. This is visible to the wallet user, and
+ *        the wallet app may have obtained it from them.
+ * @param keySource A string identifier or other metadata describing the location of the spending
+ *        key corresponding to the account's UFVK. This is set internally by the wallet app based on
+ *        its private enumeration of spending methods it supports.
+ * @param seedFingerprint The seed fingerprint. Must be length 16.
```

Comment:
This does not match "Seed fingerprint must be 32 bytes" (which indeed is the value of `JNI_ACCOUNT_SEED_FP_BYTES_SIZE`).

Reply:
Good catch. Thank you!